sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -85,107 +63,141 @@ extensions:
                  *) sudo yum-config-manager --disable $repo > /dev/null ;;
               esac
            done
-           echo "${closest_tag}" > ${jobs_repo}/ORIGIN_COMMIT
-           echo "${pkgver}" > ${jobs_repo}/ORIGIN_PKG_VERSION
-           echo "${closest_tag:1}" | cut -d'.' -f1,2 > ${jobs_repo}/ORIGIN_RELEASE
-           # disable local origin repo
            sudo yum-config-manager --disable origin-local-release > /dev/null
-           # hackety hackety hack hack - pre-releases use a *wack[y]* versioning scheme
-           if ( sudo yum install --assumeno origin${pkgver} 2>&1 || : ) | grep -q 'No package .* available' ; then
-              # just ask yum what the heck the version is
-              pkgver=$( ( sudo yum install --assumeno origin 2>&1 || : ) | awk '$1 == "x86_64" {print $2}' )
-              echo "-${pkgver}" > ${jobs_repo}/ORIGIN_PKG_VERSION
-           else
-              echo package origin${pkgver} is available
-           fi
         else
            echo Error: unknown base branch $curbranch: please resubmit PR on master or a release-x.y branch
         fi
     - type: "script"
-      title: "origin prerequisites"
-      repository: "aos-cd-jobs"
-      script: |-
-        ansible-playbook -vv --become               \
-                         --become-user root         \
-                         --connection local         \
-                         --inventory sjb/inventory/ \
-                         -e containerized=true      \
-                         -e deployment_type=origin  \
-                         /usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml
-    - type: "script"
-      title: "install origin"
-      repository: "aos-cd-jobs"
+      title: "configure system to run oc cluster up and run it"
```

Comment:
There's a lot of setup steps here that I worry will be fragile.

Reply:
How will adding more steps make it less fragile?

Reply:
Not steps as in `steps:[]` but just like -- to run `oc cluster up` correctly here we had to change a bunch of firewall rules, do all sorts of stuff, all this is logic that will need to be maintained, is all.